test/unit/tech/flash.test.js

```diff
@@ -141,3 +145,35 @@ test('seekable', function() {
   result = seekable.call(mockFlash);
   equal(result.length, mockFlash.duration_, 'seekable is empty with a zero duration');
 });
+
+// fake out the <object> interaction but leave all the other logic intact
```

Comment:
This was an experiment. It implicitly pulls in more logic from the Flash tech than the `mockFlash` mechanism used elsewhere which could be a pro or a con. I can try to convert the rest of the tests over and see if it results in less setup in the individual tests if you think it's worth considering.